docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -63,7 +63,7 @@ The binary is published on the Maven repository:
 
 - [https://central.sonatype.com/artifact/com.uid2/uid2-client](https://central.sonatype.com/artifact/com.uid2/uid2-client)
 
-## Initialization
+## Usage for DSPs
```

Comment:
Not sure about this change. Is all the content, right up to Usage for Publishers, only applicable to DSPs? Interface, Response Content, Response Statuses? At least some of that seems general. If it's all only applicable to DSPs,  not publishers, then this is fine.

Reply:
FOr publisher/sharer/advertiser/data provider, now they have their instruction section so this bit is really only for DSP now hence we propose the section name change or otherwise DSP doesn't know what to read

Reply:
@sunnywu thx.